cypress/support/commands.ts

```diff
@@ -54,10 +54,10 @@ Cypress.Commands.overwrite(
           onBeforeLoad(win) {
             options?.onBeforeLoad?.(win)
 
-            // We want to test from a clean state, so we clear the local storage (which clears redux).
-            win.localStorage.clear()
+            // We want to test from a clean state, so we clear the persisted state.
+            win.indexedDB.deleteDatabase('redux')
 
-            // Set initial user state.
+            // Since the IndexedDB state is clear, the first migration will run and pick up this legacy state from localStorage.
             win.localStorage.setItem(
               'redux_localstorage_simple_user', // storage key for the user reducer using 'redux-localstorage-simple'
               JSON.stringify({
```

Comment:
This should be setting indexedDB state, not localStorage state, as it currently acts as a migration test for the legacy state instead of an e2e test for the current state.